File: android/app/src/main/java/com/smartexpense/android/data/remote/dto/request/RegisterRequestDto.java
package com.smartexpense.android.data.remote.dto.request;

public class RegisterRequestDto {
    private String email;
    private String password;
    private String displayName;

    public RegisterRequestDto(String email, String password, String displayName) {
        this.email = email;
        this.password = password;
        this.displayName = displayName;
    }
}

File: android/app/src/main/kotlin/com/smartexpense/android/data/remote/dto/request/RegisterRequestDto.kt
package com.smartexpense.android.data.remote.dto.request

import com.google.gson.annotations.SerializedName

data class RegisterRequestDto(
    @SerializedName("email") val email: String,
    @SerializedName("password") val password: String,
    @SerializedName("displayName") val displayName: String
)
